src/content/developers/docs/gas/index.md

```diff
@@ -69,7 +69,7 @@ The base fee is calculated by a formula that compares the size of the previous b
 | 7            |          30M |        12.5% |       180.2 gwei |
 | 8            |          30M |        12.5% |       202.7 gwei |
 
-Relative to the pre-London gas auction market, this transaction-fee-mechanism change causes fee prediction to be more reliable. Following the table above - to create a transaction on block number 9, a wallet will let the user know with certainty that the **maximum base fee** to be added to the next block is `current base fee * 112.5%` or `202.8 gwei * 112.5% = 228.1 gwei`.
+Relative to the pre-London gas auction market, this transaction-fee-mechanism change causes fee prediction to be more reliable. Following the table above - to create a transaction on block number 9, a wallet will let the user know with certainty that the **minimum base fee** to be added to the next block is `current base fee * 112.5%` or `202.7 gwei * 112.5% = 228.1 gwei`.
```

Comment:
Isn't it the maximum base fee? The highest base fee a user will pay in the current block is the previous block's base fee + 12.5% (if 30M gas in that block)

Reply:
Confirmed with a few people that this is correct.

The next blocks base fee can range from a minimum of `base fee * 87.5%` to a maximum of `base fee * 112.5%`